primitives/core/src/assertion.rs

```diff
@@ -74,12 +74,19 @@ impl Assertion {
 	}
 }
 
-pub const ASSERTION_FROM_DATE: [&str; 7] = [
+pub const ASSERTION_FROM_DATE: [&str; 14] = [
 	"2017-01-01",
+	"2017-06-01",
```

Comment:
Q: Is the midpoint supposed to be `06-01` and not `07-01`? :thinking:

Reply:
good catch. Should be 07-01